packages/angular_devkit/build_angular/src/webpack/plugins/remove-hash-plugin.ts

```diff
@@ -20,25 +21,28 @@ export class RemoveHashPlugin {
 
   apply(compiler: Compiler): void {
     compiler.hooks.compilation.tap('remove-hash-plugin', compilation => {
-      const mainTemplate = compilation.mainTemplate as typeof compilation.mainTemplate & {
-        hooks: typeof compilation['hooks'];
+      const assetPath = (path: string, data: { chunk?: { name: string } }) => {
+        const chunkName = data.chunk && data.chunk.name;
+        const { chunkNames, hashFormat } = this.options;
+
+        if (chunkName && chunkNames.includes(chunkName)) {
```

Comment:
```suggestion
        if (chunkNames?.includes(chunkName)) {
```

Reply:
without `chunkName &&` it will of type `string|undefined` and `includes` will complain about it

Reply:
That's why the optional chaining operator.